backend/app/Http/Controllers/FollowController.php

```diff
@@ -2,9 +2,62 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Follow;
+use App\Models\User;
 use Illuminate\Http\Request;
 
+use function PHPUnit\Framework\isEmpty;
+
 class FollowController extends Controller
 {
-    //
+    public function index()
+    {
+        return $this->showAll(Follow::all());
+    }
+
+    public function store()
+    {
+        $data = request()->validate([
+            'follower' => ['required', 'integer'],
+            'following' => ['required', 'integer']
+        ]);
+
+        // Check if already following
+        $checkDuplicate = Follow::where([
+            ['follower_id', $data['follower']],
+            ['following_id', $data['following']]
+        ])->get();
+        if (count($checkDuplicate) > 0) {
```

Comment:
new line before condition
```suggestion
        ])->get();
        
        if (count($checkDuplicate) > 0) {
```